source/jormungandr/tests/ptref_tests.py

```diff
@@ -778,6 +778,39 @@ def test_pt_objects_with_disable_disruption(self):
         )
         assert len(response['disruptions']) == 1
 
+    def test_networks(self):
+        """test networks API"""
+        response = self.query_region("networks")
+        networks = get_not_null(response, 'networks')
+        assert len(networks) == 4
+        assert networks[0]['id'] == 'network:A'
+        assert networks[1]['id'] == 'network:B'
+        assert networks[2]['id'] == 'network:CaRoule'
+        assert networks[3]['id'] == 'network:freq'
+
+    def test_networks_with_external_code(self):
+        """test networks with external_code parameter"""
+        response = self.query_region("networks?external_code=A")
```

Comment:
minor: if it's not too much work (and not already done somewhere) you can add a request on another type for that, as this param is available for any PT collection.

Reply:
`External code` is not tested well. I can find a way to put some others API test, like lines with external_code filter.